politeiawww/user/cockroachdb/cms.go

```diff
@@ -115,6 +129,14 @@ func (c *cockroachdb) updateCMSUser(tx *gorm.DB, nu user.UpdateCMSUser) error {
 	cms := CMSUser{
 		ID: nu.ID,
 	}
+	superVisorUserIds := ""
```

Comment:
```suggestion
	var superVisorUserIds string
```

Style nit.  If you're initializing a variable to its zero value don't use a short variable declaration.  It doesn't really matter but this is how its done throughout the rest of the codebase so it should be done the same for here consistency.